main.go

```diff
@@ -107,7 +111,7 @@ func init() {
 
 func newSubnetManager() (subnet.Manager, error) {
 	if opts.kubeSubnetMgr {
-		return kube.NewSubnetManager()
+		return kube.NewSubnetManager(opts.kubeApiUrl, opts.kubeConfigFile)
```

Comment:
I notice we are not validating the URL/File path before passing it to the subnet mgr, might want to add a quick validation using `url.ParseRequestURI` and `os.Stat` so the request fails quickly on the client side instead of going higher up the stack